packages/messages/src/domains/user.js

```diff
@@ -0,0 +1,14 @@
+const { Schema } = require('mongoose');
+
+const connection = require('../drivers/mongodb/connection');
+
+const User = new Schema({
+  name: { type: String },
+  username: { type: String },
+  avatar: { type: String, required: true },
```

Comment:
Why is the avatar required? What happen if a user does not what to add an avatar to its profile?